src/tiledquickplugin/maploader.cpp

```diff
@@ -46,7 +46,9 @@ void MapLoader::setSource(const QUrl &source)
 
     Tiled::MapReader mapReader;
 
-    Tiled::Map *map = mapReader.readMap(source.toLocalFile());
+    const QString qrcScheme = QStringLiteral("qrc");
+    const bool isQrcScheme = source.scheme() == qrcScheme;
+    Tiled::Map *map = mapReader.readMap(isQrcScheme ? source.toString().mid(qrcScheme.length()) : source.toLocalFile());
```

Comment:
I don't understand. If the file to be read is a `qrc` file, you're taking the `qrc` prefix out? Why not just keep it? Or maybe you could try `source.path()` instead?